cloudbuild/python-scripts/parse_logs.py

```diff
@@ -190,6 +194,41 @@ def get_blob_and_check_metric(
             gcs_log_object, logs_as_string
         )
 
+    if mode == 'unbounded':
+        '''
+        Check if blob containing the logs contain the string of the form 
+        "Number of records processed (%d) exceed the expected count (%d)"
+        then throw an error,
+        If not get logs of type  "%d number of records have been processed".
+        Read the blob and get the metric from them,
+        and check if they match the bigquery table count.     
+        '''
+        # This is hardcoded on the basis of logs
+        records_exceed_log_pattern = (r'Number of records processed \((\d+)\) exceed the expected '
+                                      r'count \((\d+)\)')
+        match = re.search(records_exceed_log_pattern, logs_as_string)
+
+        # Check if blob containing the logs contain the string of the form
+        # "Number of records processed (%d) exceed the expected count (%d)"
+        # then throw an error.
```

Comment:
`If logs contain the string mentioned below, then count of records has exceeded the expected count, so throw an error.`